src/app/containers/IndexPageSection/utilities/filterAllowedItems.test.jsx

```diff
@@ -92,16 +92,16 @@ const numberOfStories = {
     expectedShowAllPromosAllowedItems: range(0, 13),
   },
   14: {
-    input: range(0, 14),
+    input: range(0, 13),
```

Comment:
I don't understand why this test nees to change? Isn't it intended to check how 14 and 20 items in the data are handled respectively?

Reply:
@andrewscfc Yes, you are absolutely correct. I have have amended the tests to reflect this.